charts/opensearch-dashboards/values.yaml

```diff
@@ -129,8 +129,8 @@ ingress:
       paths:
         - path: /
           backend:
-            serviceName: chart-example.local
-            servicePort: 80
+            serviceName: opensearch-dashboards
+            servicePort: 5601
```

Comment:
This will break things a little bit because you're putting a value in the `serviceName` & `servicePort`, so we'll never hit your `default $fullName` condition.

Reply:
Changed as suggested